src/commons/product_image_selection.rs

```diff
@@ -0,0 +1,347 @@
+use k8s_openapi::api::core::v1::LocalObjectReference;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+use strum::AsRefStr;
+
+pub const STACKABLE_DOCKER_REPO: &str = "docker.stackable.tech/stackable";
+
+#[derive(Clone, Debug, Deserialize, JsonSchema, PartialEq, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct ProductImage {
```

Comment:
Should this be fragmented/optional?

Reply:
Honestly, i have never used fragments yet. i would leave the decision to you. I personally am quite happy with the current solution

Reply:
It's probably not required for now...